### PR TITLE
Sort CMake installdata/treedata, add-on manifest and colors

### PR DESCRIPTION
**Pull request by garbear**
## Description

Once again I return to fend off the advancement of greying developer hair. This time by sorting some CMake definitions and addon manifest/color XML.

Something to look for in review: Lines added should equal lines deleted for all files.

## Motivation and context

Be vanquished my gray hairs! (if only)

## How has this been tested?

Test build in progress. I'll report back when I have results.

## What is the effect on users?

Younger looking developers.

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [x] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

### CODE REVIEW

File: system/dialogcolors.xml [hunk 1]
```diff
@@ -53,73 +51,96 @@
...
 	<color name="Mulberry">FFC54B8C</color>
 	<color name="Navy Blue">FF1974D2</color>
 	<color name="Neon Carrot">FFFFA343</color>
+	<color name="Ocean Foam">FF62C9D3</color>
 	<color name="Olive Green">FFBAB86C</color>
-	<color name="Orange">FFFF7538</color>
+	<color name="Orange Peel">FFFF8021</color>
 	<color name="Orange Red">FFFF2B2B</color>
 	<color name="Orange Yellow">FFF8D568</color>
+	<color name="Orange">FFFF7538</color>
```
Comment (hudokkow) [inline system/dialogcolors.xml:98]: How come `"Orange" `comes after `"Orange whatever"`?

File: system/dialogcolors.xml [hunk 2]
```diff
@@ -53,73 +51,96 @@
...
+	<color name="Raisin Black">ff212121</color>
 	<color name="Raw Sienna">FFD68A59</color>
 	<color name="Raw Umber">FF714B23</color>
 	<color name="Razzle Dazzle Rose">FFFF48D0</color>
 	<color name="Razzmatazz">FFE3256B</color>
-	<color name="Red">FFEE204D</color>
 	<color name="Red Orange">FFFF5349</color>
+	<color name="Red Satin">FF9F3434</color>
 	<color name="Red Violet">FFC0448F</color>
+	<color name="Red">FFEE204D</color>
```
Comment (hudokkow) [inline system/dialogcolors.xml:124]: Same here.

File: system/dialogcolors.xml [hunk 3]
```diff
@@ -139,28 +160,7 @@
...
-	<color name="Moonlit Pond">FF4F2CD0</color>
-	<color name="Ocean Foam">FF62C9D3</color>
-	<color name="Orange Peel">FFFF8021</color>
-	<color name="Pesto">FF5F7B4A</color>
-	<color name="Pink Luster">FFFFB2E7</color>
-	<color name="Red Satin">FF9F3434</color>
-	<color name="Sea Glass">FFC4EA7F</color>
-	<color name="Shooting Star">FFFFFF65</color>
-	<color name="Sunset Shimmer">FFF79015</color>
+	<color name="Yellow">FFFCE883</color>
```
Comment (hudokkow) [inline system/dialogcolors.xml:165]: And here.

File: system/dialogcolors.xml (hunk 1 above)
Comment (garbear) [reply system/dialogcolors.xml:98]: Sorting is lexicographic. For english letters this means alphabetical order, but for non-letters sorting is done by ASCII value.

**Comment by wsnipex** [comment]
@razzeee the change to addon-manifest.xml is going to break flatpak (again)

**Comment by razzeee** [comment]
> @razzeee the change to addon-manifest.xml is going to break flatpak (again)

Yeah, but we just need to redo the patch, then again, we were unsure if it's needed anymore anyway? Due to the other patch?

**Comment by wsnipex** [comment]
the other patch should only affect pvr, so it should still be needed

**Comment by garbear** [comment]
@razzeee are we good here?

**Comment by garbear** [comment]
It looks like everyone approves, will merge in the next few days

**Comment by garbear** [comment]
ping @razzeee for flathub beta updates

**Comment by Rechi** [comment]
Why was `cmake/treedata/darwin_embedded/tvos/tvos.txt` marked executable?

**Comment by garbear** [comment]
I git history I see:

```
commit 5b152cd2f32568e87ee8ba0b3e7f9870028dcca2
Author: Sylvain CECCHETTO <cecchetto.sylvain@me.com>
Date:   Sun Apr 28 23:47:02 2019 +0200

    [tvOS] Initial Platform Commit for Apple TVOS
    
    Initial commit for TVOS platform.

diff --git a/cmake/treedata/darwin_embedded/tvos/tvos.txt b/cmake/treedata/darwin_embedded/tvos/tvos.txt
new file mode 100755
...
```

Looks like the file was originally committed as executable, not sure why. I actually don't remember fixing the perms in this PR.

**Comment by Rechi** [comment]
Sorry, I was reading it wrong, It's good that you've removed the executable flag from the file.

**Pull request merged**